### ISSUE TITLE
Casing issue with relationship links

### ISSUE THREAD

**Issue by digitalsadhu** [digitalsadhu/loopback-component-jsonapi#66]
I'm seeing dash case being used in relationship links in a collection when the actual casing is camel casing.

In loopback

project model
```json
...
"relations": {
    "companyCreatedFor": {
      "type": "belongsTo",
      "model": "company",
      "foreignKey": "companyCreatedForId"
    }
  },
...
```

Looking at the explorer we have:
GET /projects/{id}/companyCreatedFor
GET /projects/{id}/relationships/companyCreatedFor

But in the data from `GET /projects` I'm seeing:

```
"relationships": {
  "company-created-for": {
    "links": {
      "related": "http://localhost:3000/api/projects/2/company-created-for"
    }
  }
},
```

**Comment by digitalsadhu** [comment]
Ok this is not really an issue. Just need to set the correct case type in the component settings:
```json
"loopback-component-jsonapi": {
    "restApiRoot": "/api",
    "enable": true,
    "keyForAttribute": "camelCase",
    "keyForType": "camelCase",
    "keyForRelation": "camelCase"
  }
```

**Comment by digitalsadhu** [comment]
Fixed in #69
